.github/workflows/publish-cli-bins.yaml

```
@@ -53,24 +59,26 @@ jobs:
-        env:
-          - runner: ubuntu-24.04
-            build: linux-amd64
-          - runner: macos-latest
-            build: darwin-arm64
-    runs-on: ${{ matrix.env.runner }}
+        os: [linux, darwin]
+        arch: [amd64, arm64]
+    env:
+      NAME: flightctl-${{ matrix.os }}-${{ matrix.arch }}
+    runs-on: ${{ matrix.os == 'darwin' && 'macos-latest' || 'ubuntu-24.04' }}
+    if: ${{ !(matrix.os == 'linux' && matrix.arch == 'arm64') }}
```

actionlint (1.7.4)

context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)